File: FrontEnd/src/app/shared/models/ShoppingCartItem.ts
import { Product } from "./Product";

export class ShoppingCartItem {
    public shoppingCartItemId: number;
    public shoppingCartId: number;
    public quantity: number;
    public productId: number;
    public createdDate: Date;
    public modifiedDate: Date;
    constructor() {
        this.shoppingCartItemId = 0;
        this.shoppingCartId = 0;
        this.quantity = 0;
        this.productId = 0;
        this.createdDate = new Date();
        this.modifiedDate = new Date();
    }
}

File: API/BetaCycleAPI/BetaCycleAPI/Models/ShoppingCartItem.cs
using System;
using System.Collections.Generic;

namespace BetaCycleAPI.Models;

public partial class ShoppingCartItem
{
    public int ShoppingCartItemId { get; set; }

    public int ShoppingCartId { get; set; }

    public int Quantity { get; set; }

    public int ProductId { get; set; }

    public DateTime CreatedDate { get; set; }

    public DateTime ModifiedDate { get; set; }

    public Guid Rowguid { get; set; }

    public virtual Product Product { get; set; } = null!;

    public virtual ShoppingCart ShoppingCart { get; set; } = null!;
}
